tools/envoy_headersplit/headersplit.py

```diff
@@ -0,0 +1,367 @@
+# !/usr/bin/env python3
+# Lint as: python3
+"""
+This python script can dividing monolithic mock headers
+into different mock classes. We need to remove the
+over-included head files in generated class codes and
+resolve dependencies in the corresponding Bazel files
+manually.
+"""
+
+from __future__ import print_function
+
+import argparse
+from typing import Type, List, Tuple, Dict
+import clang.cindex
+from clang.cindex import TranslationUnit, Index, CursorKind, Cursor
+
+clang.cindex.Config.set_library_path("/opt/llvm/lib")
```

Comment:
Can we file an issue in this repository to make this a flag? I suspect this won't be the correct library path on all systems (especially non-Linux ones).

Reply:
Now it load the clang library by `llvm-config` if the environment variable `LLVM_CONFIG` is set.